### ISSUE TITLE
Missing Test Case - 1. Two Sum

### ISSUE THREAD

**Issue by cuma111** [LeetCode-Feedback/LeetCode-Feedback#11322]
#### Category of the bug
- [ ] Question
- [x] Solution
- [ ] Language
- [ ] Missing Test Cases 

#### Code you used for Submit/Run operation

```
// Two Sum
class Solution {
public:
    vector<int> twoSum(vector<int> &a, int s) {

    }
};
```

**Comment by cuma111** [comment]
l

**Comment by cuma111** [comment]
l

**Comment by cuma111** [comment]
q

**Comment by cuma111** [comment]
Duplicate of #

**Comment by Rahulrpr** [comment]
closed#11322

**Comment by Rahulrpr** [comment]
closed as #11322 is completed
